cms/templates/cms/home_page.html

```diff
@@ -66,15 +66,16 @@
         <div class="col-md-12 ">
           <div class="sub-banner-section">
             <div class="sub-banner-text">
-            <h3>What is MicroMasters Credential?</h3>
+            <h3>What is the MicroMasters Credential?</h3>
```

Comment:
Actually, I updated this originally because it was a typo. But I changed "the" to "a" as a result of Anna's feedback, and Maria agreed to this. We might change it again, who knows.